src/test/java/team/asd/InternApplicationTest.java

```diff
@@ -0,0 +1,13 @@
+package team.asd;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class InternApplicationTests {
```

Comment:
@olena-syrotenko please rename class according to filename and add an empty line at the end of this file

Reply:
done